concepticondata/conceptlists/Constenla-2005-110.tsv

```diff
@@ -0,0 +1,111 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	SPANISH
+Constenla-2005-110-69	69	ASH	646	ASH	CENIZA
+Constenla-2005-110-39	39	AT	1461	AT	EN
+Constenla-2005-110-48	48	BAD	1292	BAD	MALO
+Constenla-2005-110-2	2	BARK	1204	BARK	CORTEZA
+Constenla-2005-110-66	66	BIG	1202	BIG	GRANDE
+Constenla-2005-110-37	37	BIRD	937	BIRD	AVE
+Constenla-2005-110-54	54	BITE	1403	BITE	MORDER
+Constenla-2005-110-62	62	BLACK	163	BLACK	NEGRO
+Constenla-2005-110-86	86	BLOOD	946	BLOOD	SANGRE
+Constenla-2005-110-57	57	BONE	1394	BONE	HUESO
+Constenla-2005-110-6	6	BREAST	1402	BREAST	TETAS
+Constenla-2005-110-38	38	CHILD	2099	CHILD	NIÑO
+Constenla-2005-110-93	93	CLEAN	704	CLEAN	LAVAR
```

Comment:
CLEAN refers to the property. Depending on the exact meaning, this should be mapped to CLEAN (SOMETHING), WASH, or LAUNDER (WASH CLOTHES). We have thus far mapped lavar to WASH.